x-pack/plugins/maps/public/shared/components/no_index_pattern_callout.js

```diff
@@ -8,24 +8,45 @@ import chrome from 'ui/chrome';
 
 import React from 'react';
 import { EuiCallOut, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
 
 export function NoIndexPatternCallout() {
   return (
     <EuiCallOut
-      title="Couldn't find any index patterns with geospatial fields"
+      title={
+        i18n.translate('xpack.maps.noIndexPattern.messageTitle', {
+          defaultMessage: `Couldn't find any index patterns with geospatial fields`
+        })
+      }
       color="warning"
     >
       <p>
-        You&rsquo;ll need to{' '}
+        <FormattedMessage
+          id="xpack.maps.noIndexPattern.doThisDescription"
+          defaultMessage="You'll need to "
+        />
         <EuiLink href={chrome.addBasePath('/app/kibana#/management/kibana/index_pattern')}>
-          create an index pattern
-        </EuiLink>{' '}
-        with geospatial fields.
+          <FormattedMessage
+            id="xpack.maps.noIndexPattern.doThisLinkTextDescription"
+            defaultMessage="create an index pattern"
+          />
+        </EuiLink>
+        <FormattedMessage
+          id="xpack.maps.noIndexPattern.doThisWithDescription"
```

Comment:
and then rename this id to `lets rename id to `xpack.maps.noIndexPattern.doThisDescriptionSuffix`

Reply:
flipped to `xpack.maps.noIndexPattern.xpack.maps.noIndexPattern.doThisSuffixDescription`, so all end on `description` per guidelines